### PR TITLE
NATIVE-62: Fix webview width

### PR DESCRIPTION
**Pull request by NKlug**

### CODE REVIEW

File: src/modules/common/components/Page.js [hunk 1]
```diff
@@ -120,6 +133,7 @@ class Page extends React.Component<PropType, StateType> {
 
             onMessage={this.onMessage}
             style={{height: height}}
+            width={width}
```
Comment (Schedulaar) [inline src/modules/common/components/Page.js]: try to use the style attribute here for consistency

File: src/modules/common/components/Page.js [hunk 2]
```diff
@@ -46,15 +47,26 @@ type PropType = {
 }
 
 class Page extends React.Component<PropType, StateType> {
+  // noinspection JSDuplicatedDeclaration
   onMessage: (event: WebViewMessageEvent) => void
+  // noinspection JSDuplicatedDeclaration
```
Comment (Schedulaar) [inline src/modules/common/components/Page.js]: try to avoid these with arrow-functions.

File: src/modules/common/components/Page.js (hunk 1 above)
Comment (NKlug) [reply src/modules/common/components/Page.js]: Done.

File: src/modules/common/components/Page.js (hunk 2 above)
Comment (NKlug) [reply src/modules/common/components/Page.js]: Done.

**Pull request merged**